experimental/nodejs-functions/stack.yaml

```diff
@@ -7,6 +7,9 @@ maintainers:
   - name: Chris Bailey
     email: cnbailey@gmail.com
     github-id: seabaylea
+  - name: Sam Roberts
+    email:
```

Comment:
@sam-github I've added you as a maintainer for this stack could you double check the info and add you email please

Reply:
`vieuxtech@gmail.com`